audit_report_page.py

```diff
@@ -8,10 +8,7 @@
 from docx.enum.text import WD_PARAGRAPH_ALIGNMENT
 from transcript import Transcript
 from audit_report import AuditReport
-from track import Track, ComputerScience, DataScience, IntelligentSystems, Systems
-# import transcript
-# import track
-
+from track import Track, ComputerScience, DataScience, IntelligentSystems, Systems, Traditional, InteractiveComputing, CyberSecurity, SoftwareEngineering, Network

```

Comment:
The "Track" part of from track import Track can't be included because you got rid of that when you patched audit_report.py